test/blocks/app-ratings/app-ratings.test.js

```diff
@@ -11,23 +11,27 @@ await import(`${getLibs()}/utils/utils.js`).then((mod) => {
   mod.setConfig(conf);
 });
 const { default: decorate } = await import('../../../express/code/blocks/app-ratings/app-ratings.js');
-const body = await readFile({ path: './mocks/body.html' });
-document.body.innerHTML = body;
+
 describe('App Ratings', () => {
-  before(() => {
+  before(async () => {
+    const body = await readFile({ path: './mocks/body.html' });
+    document.body.innerHTML = body;
     window.isTestEnv = true;
     window.placeholders = {
```

Comment:
if you are building up states that could affect other tests then it's better to keep it in beforeEach() or do a cleanup in after()